### DISCUSSION TITLE
Display custom objects on map

### DISCUSSION THREAD

**Discussion by magdel** [organicmaps/organicmaps#9142] [Ideas]
Hello!
We want to display locations of objects on map. Locations are obtained via BLE by custom protocol.
Is there any standart way to extend, say standart Android application, to display some points and description to it? 
OrganicMaps are offline and location data is also from offline trackers, so we see great possibilities with integration.
May be there is already some work is done in displaying trackers on map? Or only way is to fork?

Just want to switch from the app: https://github.com/Udj13/AGLoRa-client-flutter

Thank you, any help is appreciated.

**Comment by biodranik** [comment]
Please check examples at https://omaps.app/api https://github.com/organicmaps/api-android and https://github.com/organicmaps/api-ios

**Reply by biodranik** [reply]
And please don't hesitate to ask questions and offer improvements/fixes, we would be happy to make OM's API more useful.

**Reply by magdel** [reply]
Thank you! Looks great!
